src/utils/response/rawApiResponse.ts

```diff
@@ -0,0 +1,33 @@
+import { AxiosResponse } from 'axios';
+
+export type RawApiResponse = {
```

Comment:
This looks just like ApiResponse.ts or response.ts. I think we should think about merging them

Reply:
No, it is not. and it is needed for deltaSync because I new two different types for the results of the deltaSync